app/views/projects/samples/attachments/_attachment.html.erb

```diff
@@ -2,25 +2,27 @@
   id="<%= dom_id(attachment) %>"
   class="bg-white border-b dark:bg-slate-800 dark:border-slate-700"
 >
-  <td class="px-6 py-4">
-    <%= check_box_tag "attachment_ids[]",
-    (
-      if attachment.associated_attachment
-        [attachment.id, attachment.associated_attachment.id].to_s
-      else
-        attachment.id
-      end
-    ),
-    nil,
-    id: dom_id(attachment.file, :file),
-    "aria-label": attachment.file.filename,
-    data: {
-      selection_target: "rowSelection",
-      action: "input->selection#toggle"
-    },
-    class:
-      "w-4 h-4 text-primary-600 bg-slate-100 border-slate-300 rounded focus:ring-primary-500 dark:focus:ring-primary-600 dark:ring-offset-slate-800 focus:ring-2 dark:bg-slate-700 dark:border-slate-600" %>
-  </td>
+  <% if allowed_to?(:update?, @project) %>
```

Comment:
I think we should use the policy method `update_sample?` instead as the attachments belong to a sample

Reply:
Updated in e75c67e5f4998e09d5e36342368a13419a860e91